app/src/androidTest/java/net/skyscanner/backpack/button/BpkButtonTest.kt

```diff
@@ -55,7 +60,7 @@ class BpkButtonTest(flavour: Flavor) : BpkSnapshotTest() {
     // we want to see colors of all types
     // different sizes have different text style
 
-    capture {
+    capture(background = type.rowBackground()) {
```

Comment:
set the background in tests - this is for the on dark/light buttons to make sure they are visible correctly